connector/scrapers/kutis_scraper.py

```
@@ -0,0 +1,130 @@
+                )
+            else:
+                content = await self._fetch(HTTPMethod.GET, KUTIS_TIMETABLE_PAGE_URL)
+
+            tables = content.find_all("table", class_="list06")
+            days = ["월요일", "화요일", "수요일", "목요일", "금요일", "토요일"]
+            classes: list[dict] = []
+
+            if len(tables) == 3:
+                timetables = [tables[1], tables[2]]
+            else:
+                timetables = [tables[1]]
```

Ruff (0.11.9)

Use ternary operator `timetables = [tables[1], tables[2]] if len(tables) == 3 else [tables[1]]` instead of `if`-`else`-block

Replace `if`-`else`-block with `timetables = [tables[1], tables[2]] if len(tables) == 3 else [tables[1]]`

(SIM108)